yjit/src/disasm.rs

```diff
@@ -175,13 +175,20 @@ pub fn disasm_addr_range(cb: &CodeBlock, start_addr: usize, end_addr: usize) ->
 
     // For each instruction in this block
     for insn in insns.as_ref() {
+        // Colorize outlined code in blue
+        let (prefix, suffix) = if cb.outlined {
+            ("\x1b[34m", "\x1b[0m")
+        } else {
+            ("", "")
+        };
+
         // Comments for this block
         if let Some(comment_list) = cb.comments_at(insn.address() as usize) {
             for comment in comment_list {
-                writeln!(&mut out, "  \x1b[1m# {}\x1b[0m", comment).unwrap();
+                writeln!(&mut out, "  \x1b[1m{prefix}# {comment}{suffix}\x1b[0m").unwrap();
             }
         }
-        writeln!(&mut out, "  {}", insn).unwrap();
+        writeln!(&mut out, "  {prefix}{insn}{suffix}").unwrap();
```

Comment:
Do you have to switch color for every single line or can you do it when the block begins and ends?

Reply:
I wasn't sure if you can disable bold while leaving blue, but it turns out it's possible with `\x1b[22m`. done https://github.com/ruby/ruby/pull/7073/commits/1b1f0f03dda19dff720c8d74baaca39f963fef8b

Reply:
I had wondered the same, but often it's better to start and end often. It means if output gets intermixed (e.g. with a background thread or process) then you won't wind up with accidental big areas of bold/coloured/standard/un-coloured when you didn't want them.

It's probably fine either way here.